components/twitter/twitter.app.js

```diff
@@ -973,6 +973,27 @@ module.exports = {
         }
       })).data
     },
+    async getUserTimeline(opts = {}) {   
+      const {
+        screen_name,
+        count = 100,
+        exclude_replies,
+        include_rts,
+        since_id,
+      } = opts
+      return (await this._makeRequest({
+        url: `https://api.twitter.com/1.1/statuses/user_timeline.json`,
```

Comment:
Do you abstract the API base URL (`https://api.twitter.com/1.1`) elsewhere? Might be good to have that defined in one place.

Reply:
good point. i'll make that change in a future update.